dcc-portal-ui/app/scripts/oncogrid/js/directive.js

```diff
@@ -268,7 +268,7 @@
             donors: donors,
             genes: genes,
             observations: observations,
-            element: '#oncogrid-div',
+            element: $element.find('#oncogrid-div').get(0),
```

Comment:
Initially tried to refactor and cache everything at the top of the directive's link function in `$gridContainer`, `$gridSpinner`, `$gridButton` etc, but selections were coming up empty. 

Not to excuse uncached selections, but I think that's probably why they were being used exclusively [throughout the directive](https://github.com/icgc-dcc/dcc-portal/blob/9b50a6d879f6d1804c00ce6014d3eb3995e02922/dcc-portal-ui/app/scripts/oncogrid/js/directive.js#L298-L308) and [in the controller as well](https://github.com/icgc-dcc/dcc-portal/blob/9b50a6d879f6d1804c00ce6014d3eb3995e02922/dcc-portal-ui/app/scripts/oncogrid/js/controller.js#L75-L77)

I think selectively caching some parts would make things messier. My thought was to minimize the diff for this hotfix and we can refactor this directive + controller into a component instead